<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreshScore</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/logo.ico</normaloff>:/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #243434; color:white; </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="lblScore">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>96</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>62</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #2B2B2B; color:white; border-color:black; </string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btnLoss">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: grey;</string>
       </property>
       <property name="text">
        <string>LOSS</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lblWin">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>64</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #2B2B2B; color:white;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblLoss">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>64</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #2B2B2B; color:white;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnWin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: grey;</string>
       </property>
       <property name="text">
        <string>WIN</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblTopGoal">
       <property name="maximumSize">
        <size>
         <width>640</width>
         <height>18</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white; background-color:#2B2B2B;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Goal: 10</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QProgressBar" name="progressBar">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar:horizontal {
border: 1px solid gray;
border-radius: 3px;
background: black;
padding: 1px;
text-align: right;
margin-right: 50%;
}
QProgressBar::chunk:horizontal {
background: #045A04;
margin-right: 1px; /* space */
width: 5px;
}</string>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:grey;</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black; color: white;</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSet_Goal">
    <property name="styleSheet">
     <string notr="true">background-color:black; color: white;</string>
    </property>
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionSet_Goal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSet_Goal"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Clear all fields</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSet_Goal">
   <property name="text">
    <string>Set Goal</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
